playwright/tests/standalone/key-listener/quick-action.spec.ts

```diff
@@ -85,24 +85,3 @@ test('wrap, jump and unwrap', async ({ page, browserName }) => {
   await expect(end.locator()).toBeVisible();
   await expect(embedded.locator()).toBeHidden();
 });
-
-test('create node', async ({ page }) => {
```

Comment:
why remove this test?

Reply:
As far as I can tell, the Palette components from the `ui-component` library do not have keyboard navigation implemented and the shortcut handling is already covered by the test above. I can look into restoring the keyboard navigation behavior but long-term it should probably go into the ui-components library.

Reply:
This is correct, however technically if the focus is correctly set then you can navigate via tab.
I quickly tried the current state and one problem is that the search is not in focus when this palette opens. I think this would be good to have again. 
Because this is one of your "quick edit" modes. select an element -> press "a" -> type for search -> press enter to insert the first element (or better the one which was selected via the arrow keys, but now I think it is fine if it is the first element otherwise you can navigate with tab).

And at least the second part was testing this...

Reply:
I pushed another update. However, there is no good hook in the palette to have the focus on the search (for typing) but at the same time have one of the items marked active (in case of 'Enter'). So we just have the typical keyboard navigation now. I adapted the test case accordingly, please let me know if that is alright with you.